.github/workflows/delete_review_app.yml

```diff
@@ -39,28 +39,25 @@ jobs:
         fi
         echo "PR_NUMBER=$PR_NUMBER" >> $GITHUB_ENV
```

Comment:
it needs 
echo "ENVIRONMENT=pr-${PR_NUMBER}" >> $GITHUB_ENV
after line 40 as it's used in L63